contrib/hugin.lua

```diff
@@ -38,35 +40,54 @@ local df = require "lib/dtutils.file"
 require "official/yield"
 local gettext = dt.gettext
 
+local namespace = 'module_hugin'
+local user_pref_str = 'prefer_gui'
+local user_prefer_gui = dt.preferences.read(namespace, user_pref_str, "bool")
+local hugin_widget = nil
+local exec_widget = nil
+local executable_table = {"hugin", "hugin_executor", "pto_gen"}
+
 -- works with darktable API version from 2.0.0 to 5.0.0
 dt.configuration.check_version(...,{2,0,0},{3,0,0},{4,0,0},{5,0,0})
```

Comment:
Once we make the changes to run on windows, specifically using dt.configuration.running_os or dtutils.check_os(), then the API version that this works with is only 5.0.0+